src/sagas/reown.js

```diff
@@ -744,22 +940,20 @@ export function* featureToggleUpdateListener() {
  * Sends a disconnect session RPC message to the connected cloud server
  */
 export function* onCancelSession(action) {
-  const { web3wallet } = yield select((state) => state.walletConnect.client);
+  const { walletKit } = yield select((state) => state.reown.client);
 
-  const activeSessions = yield call(() => web3wallet.getActiveSessions());
+  const activeSessions = yield call(() => walletKit.getActiveSessions());
 
-  if (!activeSessions[action.payload]) {
-    return;
+  if (activeSessions[action.payload.id]) {
```

Comment:
Is this `action.payload` vs `action.payload.id` a difference in the `web3wallet` and `walletKit` packages?

Reply:
I think this was a bug in the code actually